images.yaml

```diff
@@ -175,6 +175,10 @@
 - name: golang
   patterns:
   - pattern: '>= 1.13.1'
+- name: golangci/golangci-lint
+  tags:
+  - sha: 757ee77f748bb5d93d584a9ab5710f4149adcdacf2f3584c14b42015f7811b64
+    tag: v1.23.6
```

Comment:
Hold on - why not use a pattern? Pretty sure this evolves with the go version, right?

Reply:
Following my own advice and not using a pattern until there is some automation to actually use a new image -- stay tuned